### PR TITLE
Update solr to 9.4.1

### PR DESCRIPTION
**Pull request by kokosing**
Update solr to 9.4.1

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -200,7 +200,7 @@
         <servlet.api.version>2.5</servlet.api.version>
         <slf4j-api.version>2.0.13</slf4j-api.version>
         <slf4j.version>2.0.13</slf4j.version>
-        <solr.version>8.11.2</solr.version>
```
Comment (mneethiraj) [inline pom.xml:203]: Build fails with Solr version updated to 9.4.1.  Solr 8.11.3 is the last supported version in JDK8. Higher versions require JDK11.

**Comment by mneethiraj** [comment]
@kokosing  - Solr version update to 8.11.3 is merged in master and ranger-2.5 branches, via RANGER-4870. I suggest tracking the config cleanup in a separate PR. Thank you!

**Comment by kokosing** [comment]
Thank you for merging and feedback!

**Comment by kokosing** [comment]
See https://github.com/apache/ranger/pull/358

File: pom.xml (hunk 1 above)
Comment (kokosing) [reply pom.xml:203]: > Build fails with Solr version updated to 9.4.1. Solr 8.11.3 is the last supported version in JDK8. Higher versions require JDK11.

I noticed that and I couldn't figure out why. In IDE it was working just fine, but not in CLI.

**Pull request closed without merging**